### PR TITLE
Working through chapter four

### PR DESCRIPTION
**Pull request by NJosten99**
Chapter four should be complete.

### CODE REVIEW

**Review by jbanahan — APPROVED** [review]
It's fine for our use, but you'll want to keep changes for different purposes in separate branches/pull requests. I would suggest closing #3 and merging this one with the master branch.

File: book/2-begin/server/models/User.js [hunk 1]
```diff
@@ -1,41 +0,0 @@
-const mongoose = require('mongoose');
```
Comment (jbanahan) [inline book/2-begin/server/models/User.js:1]: Whoops did we mean to remove the work done for chapter 2?

File: book/2-end/server/server.js [hunk 2]
```diff
@@ -1,79 +1,14 @@
-const express = require('express');
```
Comment (jbanahan) [inline book/2-end/server/server.js:1]: Probably didn't mean to remove this file also?

File: book/2-begin/server/models/User.js (hunk 1 above)
Comment (NJosten99) [reply book/2-begin/server/models/User.js:1]: I did not!  Not sure what happened here.  Thank you for pointing this out.

File: book/2-end/server/server.js (hunk 2 above)
Comment (NJosten99) [reply book/2-end/server/server.js:1]: Not sure what happened here either.

**Pull request merged**